geopandas/tools/sjoin.py

```diff
@@ -30,6 +32,14 @@ def sjoin(left_df, right_df, how='inner', op='intersects',
     """
     import rtree
 
+    if not isinstance(left_df, GeoDataFrame):
+        raise ValueError("`left_df` should be GeoDataFrame, got %s" %
```

Comment:
Minor formatting question: can you use normal single quotes around 'left_df' instead of the backticks?

(same for one below)

Reply:
Also, can you use `.format(..)` instead of `%s` ? 
(I know the ones below use `%s`, but for newly added code I prefer `format`)